cirq/linalg/states.py

```diff
@@ -38,3 +38,20 @@ def one_hot(*,
     result = np.zeros(shape=shape, dtype=dtype)
     result[index] = 1
     return result
+
+
+def eye_tensor(*, qid_shape: Tuple[int, ...],
```

Comment:
Could rename from `qid_shape` to `input_shape` or `output_shape` or `half_shape`.

Don't force keyword args on the shape, to act more like numpy.

Reply:
Done.